### DISCUSSION TITLE
updown plugin Radius Attributes shaper

### DISCUSSION THREAD

**Discussion by chodorenko** [strongswan/strongswan#1299] [General]
Hello All
Please say howe i can add  Radius Attributes on script updown ?

I need create shaper for clients  by radius attributes  Ascend-Data-Rate, Ascend-Xmit-Rate and not understand where i can get its value for write UPDOWN script.

ps: May be present different path for shape users ?

**Comment by Thermi** [comment]
you can't, it's not implemented.

**Comment by ilyacherevkov** [comment]
Does it work now? Im sending a CoA request from the backend and, on reconnect, the updown script queries the database to get the attributes.  But it would be much easier if attributes were accessible in updown script directly.
